c7n/resources/route53.py

```diff
@@ -497,7 +497,9 @@ def process(self, resources, event=None):
             logging = zid in enabled_zones
             if logging and state:
                 r['c7n:log-config'] = enabled_zones[zid]
-                log_group_name = r['c7n:log-config']['CloudWatchLogsLogGroupArn'].split(":")[-1]
+                log_group_name = r['c7n:log-config']['CloudWatchLogsLogGroupArn']\
+                    .split("log-group:")[-1]
+                log_group_name = log_group_name.replace(':*', '')
```

Comment:
nit: if we've already split the arn, it might be easier/clearer to go from grabbing "the last segment" to "the sixth segment". that way we'll ignore the `*` log stream specifier if it exists, without having to run a replace.

(the replace seems fine here so i call this a nit, just felt that "changing more than we need to" spidey sense tingling) :grin:

Reply:
@ajkerrigan thank you - agreed your recommendation is cleaner, I've updated the code accordingly, please re-review when you have a chance